src/camera.cpp

```diff
@@ -453,7 +453,7 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 
 	// Get FOV
 	f32 fov_degrees;
-	if (player->getPlayerControl().zoom && m_client->checkLocalPrivilege("zoom")) {
+	if (player->getPlayerControl().zoom && player->can_zoom) {
```

Comment:
can you make can_zoom private and use setters/getter ? Those public attributes in Player class are very anoying

Reply:
I'll try.

Reply:
Done.